tekton/ci/jobs/tekton-github-tasks-completed.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: github-tasks-completed
+  namespace: tektonci
+  description: |
+    Verifies that a PR has all the tasks completed
```

Comment:
Tiny nit: suggest expanding this a bit to describe what we consider a task to be. Something like `Verifies that a PR has all the tasks completed. A task is considered to be any part of the text that looks like a github markdown checkbox ("- []")`.

Reply:
much clearer description - updated it - thanks :)